src/api/constants/api-constants.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ *   Copyright (c) 2024 WSO2 Inc. (http://www.wso2.org)
+ *   All rights reserved.
+ *   
+ *   This software is the property of WSO2 Inc. and its suppliers, if any.
+ *   Dissemination of any information or reproduction of any material contained
+ *   herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+ *   You may not alter or remove any copyright or other notice from copies of this content.
+ */
+
+/**
+ * Class containing API request method constants.
+ */
+export class RequestMethodTypes {
```

Comment:
These two can be an enums.

Move it under `src/api/models/requests.ts` file.

Reply:
Fixed with 735907b4616df68c4c072a72121104c1b697ba76